modules/nw-route-specific-annotations.adoc

```diff
@@ -54,3 +55,33 @@ metadata:
 Setting a server-side timeout value for passthrough routes too low can cause
 WebSocket connections to timeout frequently on that route.
 ====
+
+
+.A route rewrite-target annotation
+[source,yaml]
+----
+apiVersion: v1
+kind: Route
+metadata:
+  annotations:
+    haproxy.router.openshift.io/rewrite-target: / <1>
+...
+----
+<1> Sets / as rewrite path of the request on the backend
+
+.rewrite-target examples:
+[cols="4*", options="header"]
+|===
+|Route.spec.path|Request path|Rewrite target| Output
+|/foo|/foo|/|/
+|/foo|/foo/|/|/
+|/foo|/foo/bar|/|/bar
+|/foo|/foo/bar/|/|/bar/
+|/foo|/foo|/bar|/bar
+|/foo|/foo/|/bar|/bar/
+|/foo|/foo/bar|/baz|/baz/bar
+|/foo|/foo/bar/|/baz|/baz/bar/
+|/foo/|/foo|/|Application is not available (Error 404)
```

Comment:
In this example, the request does not use the route, right?  But the request could use another route, if there is some route with the same host but with `spec.path` omitted or set to `/`, so the response will not necessarily be a 404.  Better to say something along the lines of, "N/A (request path does not match route path)".